Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml

```diff
@@ -163,6 +169,8 @@
   <remove-node path="/api/package[@name='kotlin.text']/class[@name='StringsKt__StringsKt']/method[@name='trimStart' and count(parameter)=2 and parameter[1][@type='java.lang.String'] and parameter[2][@type='char...']]" />
   <remove-node path="/api/package[@name='kotlin.text']/class[@name='StringsKt__StringsKt']/method[@name='trimStart' and count(parameter)=2 and parameter[1][@type='java.lang.String'] and parameter[2][@type='kotlin.jvm.functions.Function1&lt;? super java.lang.Character, java.lang.Boolean&gt;']]" />
 
+  <remove-node path="/api/package[@name='kotlin.time']/class[@name='Duration']/method[@name='div-LRDsOJo' and count(parameter)=2 and parameter[1][@type='double'] and parameter[2][@type='double']]" />
```

Comment:
This is a special case where the second parameter is really a `Duration` - a magical type that is a Kotlin-ified `double`. Just like how `ULong` is a `ulong` and you can't get one.